build/docker-compose.yml

```diff
@@ -3,6 +3,7 @@ version: "3.98"
 services:
   web:
     container_name: web
+    image: ghcr.io/tbd54566975/ssi-service:main
     build:
```

Comment:
without deleting the build blocks here, it will still try to build it with the defined dockerfile even you added an "image" here. 
That is the case on my computer, just FYI

Reply:
hrm - maybe it didn't on mine as it had already built it previously?

Reply:
ok may just need a different compose file for running it from images